### DISCUSSION TITLE
Data push : DynamoDB to IDS

### DISCUSSION THREAD

**Discussion by iamynr5** [International-Data-Spaces-Association/DataspaceConnector#87] [Q&A] [answered]
Currently my data's are in DynamoDB tables  how to push that data to IDS dataconnector

If possible can you provide the steps so that it will be helpful for me to achieve this

**Comment by HeinrichPet** [comment, marked as answer]
Hi,

here is a step-by-step manual to provide data in the IDS: https://international-data-spaces-association.github.io/DataspaceConnector/CommunicationGuide/v6/Provider
